library/src/main/java/com/pokegoapi/api/settings/CatchOptions.java

```diff
@@ -198,10 +198,8 @@ public Pokeball getItemBall(double encounterProbability) throws LoginFailedExcep
 	 */
 	public int getRazzberries() {
 		int razzberries = maxRazzBerries;
```

Comment:
Is that your final suggestion? It is rather tedious to attempt to adjust as the comments flood in.